src/CppUTest/JUnitTestOutput.cpp

```diff
@@ -135,6 +136,9 @@ void JUnitTestOutput::printCurrentTestStarted(const UtestShell& test)
         impl_->results_.tail_ = impl_->results_.tail_->next_;
     }
     impl_->results_.tail_->name_ = test.getName();
+    if (*(test.getProgressIndicator()) == '!') {
```

Comment:
Could you change it still? This way the JUnitOutput would break down if we change the ignored indicator on the command line. I'd say that is a weird side-effect :)